_data/index.yml

```diff
@@ -36,6 +45,15 @@ apps:
         - path: designing-your-journal
         - path: gdpr
         - path: translation
+- path: sust-disc
```

Comment:
@rcgillis Similar to the other comment, this should be named based on a card in _includes/cards/omp3. It should also be indented to align with the list above it. Other elements below this (lines 49-56) are not needed for this file as they are found in the card file.